### ISSUE TITLE
globalwarmingpotentials needs update for pydantic>=2.0.0

### ISSUE THREAD

**Issue by MichaelTiemannOSC** [openclimatedata/globalwarmingpotentials#24]
The 2.x version of Pydantic introduces many new features and many new deprecations.  One such is this:

```
../../../../virtualenvs/itr-pydantic/lib/python3.11/site-packages/globalwarmingpotentials/__init__.py:26
  ~/virtualenvs/itr-pydantic/lib/python3.11/site-packages/globalwarmingpotentials/__init__.py:26: DeprecationWarning:
  
  open_text is deprecated. Use files() instead. Refer to https://importlib-resources.readthedocs.io/en/latest/using.html#migrating-from-legacy for migration advice.
```

Thanks!

**Comment by znicholls** [comment]
Thanks. If you can see the trivial fix, PRs welcome :)

**Comment by MichaelTiemannOSC** [comment]
Here's a trivial PR: https://github.com/openclimatedata/globalwarmingpotentials/pull/25

**Comment by rgieseke** [comment]
Released as 0.9.4, thanks for the fix!
